docs/user_docs/observability/monitor-database.md

```diff
@@ -10,231 +10,196 @@ import TabItem from '@theme/TabItem';
-4. Open and view the web console of a monitoring dashboard. For example,
+Create a new cluster with the following command, ensuring the monitoring exporter is enabled.
 
-   ```bash
-   kbcli dashboard open kubeblocks-grafana
-   ```
+:::note
 
-</TabItem>
+Make sure `spec.componentSpecs.disableExporter` is set to `false` when creating a cluster.
 
-</Tabs>
+:::
 
-## For production environment
+```yaml
+cat <<EOF | kubectl apply -f -
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: Cluster
+metadata:
+  name: mycluster
+  namespace: demo
+spec:
+  clusterDefinitionRef: postgresql
+  clusterVersionRef: postgresql-12.14.0
+  terminationPolicy: Delete
+  affinity:
+    podAntiAffinity: Preferred
+    topologyKeys:
+    - kubernetes.io/hostname
+    tenancy: SharedNode
+  tolerations:
+    - key: kb-data
+      operator: Equal
+      value: 'true'
+      effect: NoSchedule
+  componentSpecs:
+  - name: postgresql
+    componentDefRef: postgresql
+    enabledLogs:
+    - running
+    disableExporter: true # Set to `false` to enable exporter
+    replicas: 2
+    resources:
+      limits:
+        cpu: '0.5'
+        memory: 0.5Gi
+      requests:
+        cpu: '0.5'
+        memory: 0.5Gi
+    volumeClaimTemplates:
+    - name: data
+      spec:
+        accessModes:
+        - ReadWriteOnce
+        resources:
+          requests:
+            storage: 20Gi
+EOF
+```
 
-For the production environment, it is highly recommended to build your monitoring system or purchase a third-party monitoring service.
+#### For an existing cluster
 
-### Integrate dashboard and alert rules
+If a cluster already exists, you can run the command below to verify whether the monitoring exporter is enabled.
 
-Kubeblocks provides Grafana Dashboards and Prometheus AlertRules for mainstream engines, which you can obtain from [the repository](https://github.com/apecloud/kubeblocks-mixin), or convert and customize according to your needs.
+```bash
+kubectl get cluster mycluster -o yaml
+```
 
-For the importing method, refer to the tutorials of your third-party monitoring service.
+View the output.
 
-### Enable the monitoring function for a database
+```yaml
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: Cluster
+metadata:
+...
+spec:
+   ...
+   componentSpecs:
+   ...
+      disableExporter: false
+```
 
-Check whether the monitoring function of the cluster is enabled. If the monitoring function is enabled, the output shows `disableExporter: false`.
+If `disableExporter: false` is not shown in the output, it means the monitoring function of this cluster is not enabled and you need to enable it first.
 
 <Tabs>
 
-<TabItem value="kubectl" label="kubectl" default>
-
-If you disable the monitoring function when creating a cluster, run the command below to enable it.
+<TabItem value="kubectl patch" label="kubectl patch" default>
 
 ```bash
 kubectl patch cluster mycluster -n demo --type "json" -p '[{"op":"add","path":"/spec/componentSpecs/0/disableExporter","value":false}]'
```

Comment:
update disableExporter will restart all pods in the cluster. should highlight the impact.